docker/Dockerfile

```diff
@@ -28,47 +28,7 @@ ARG GID=1000
 
 # PDH and WEL extensions and not listed as they are Windows specific
 # SYSTEMD extension is turned OFF explicitly as it has no use in an alpine container
-ARG ENABLE_ALL=OFF
-ARG ENABLE_PYTHON=OFF
-ARG ENABLE_OPS=ON
-ARG ENABLE_JNI=OFF
-ARG ENABLE_OPENCV=OFF
-ARG ENABLE_OPC=OFF
-ARG ENABLE_GPS=OFF
-ARG ENABLE_COAP=OFF
-ARG ENABLE_SQL=OFF
-ARG ENABLE_MQTT=OFF
-ARG ENABLE_PCAP=OFF
-ARG ENABLE_LIBRDKAFKA=OFF
-ARG ENABLE_SENSORS=OFF
-ARG ENABLE_USB_CAMERA=OFF
-ARG ENABLE_TENSORFLOW=OFF
-ARG ENABLE_AWS=OFF
-ARG ENABLE_BUSTACHE=OFF
-ARG ENABLE_SFTP=OFF
-ARG ENABLE_OPENWSMAN=OFF
-ARG ENABLE_AZURE=OFF
-ARG ENABLE_ENCRYPT_CONFIG=ON
-ARG ENABLE_NANOFI=OFF
-ARG ENABLE_SPLUNK=OFF
-ARG ENABLE_GCP=OFF
-ARG ENABLE_ELASTICSEARCH=OFF
-ARG ENABLE_TEST_PROCESSORS=OFF
-ARG DISABLE_CURL=OFF
-ARG DISABLE_JEMALLOC=ON
-ARG DISABLE_CIVET=OFF
-ARG DISABLE_EXPRESSION_LANGUAGE=OFF
-ARG DISABLE_ROCKSDB=OFF
-ARG DISABLE_LIBARCHIVE=OFF
-ARG DISABLE_LZMA=OFF
-ARG DISABLE_BZIP2=OFF
-ARG ENABLE_SCRIPTING=OFF
-ARG DISABLE_PYTHON_SCRIPTING=
-ARG ENABLE_LUA_SCRIPTING=
-ARG ENABLE_KUBERNETES=OFF
-ARG ENABLE_PROCFS=OFF
-ARG ENABLE_PROMETHEUS=OFF
-ARG DISABLE_CONTROLLER=OFF
+ARG MINIFI_OPTIONS=""
```

Comment:
Maybe we could add a comment what this argument should look like if someone would want to use this dockerfile individually without using the shell script or a make target.